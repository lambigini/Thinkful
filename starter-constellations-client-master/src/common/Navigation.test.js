import React from "react";
import Navigation from "./Navigation";
import renderer from "react-test-renderer";

it("renders the Navigation", () => {
  const component = renderer.create(<Navigation />);
  expect(component.toJSON()).toMatchSnapshot();
});
