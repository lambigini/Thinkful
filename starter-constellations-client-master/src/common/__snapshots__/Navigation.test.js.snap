// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navigation 1`] = `
<ul
  className="nav justify-content-center bg-light py-2"
>
  <li
    className="nav-item"
  >
    <button
      className="btn btn-link nav-link active"
    >
      Home
    </button>
  </li>
  <li
    className="nav-item"
  >
    <button
      className="btn btn-link nav-link"
    >
      About
    </button>
  </li>
</ul>
`;
